app/interactions/pulitzer/update_version_status.rb

```diff
@@ -22,10 +22,14 @@ def make_version_active
 
   def make_version_abandoned
     @active_version = @transitional_version.post.active_version
-    @transitional_version.update(status: :abandoned)
-    @processing_version = @post.create_processing_version
-    Pulitzer::CloneVersionJob.perform_later(@active_version)
-    @processing_version
+    if @active_version == @transitional_version
+      @transitional_version.update(status: :abandoned)
```

Comment:
@gomayonqui This line could come before the if so you don't have to repeat it in the else.